### ISSUE TITLE
Cover no longer added

### ISSUE THREAD

**Issue by IanBlakeley** [slhck/ffmpeg-normalize#246]
**Checklist** (please tick all boxes)
- [x] I am using the latest version of `ffmpeg-normalize` (run `pip3 install --upgrade ffmpeg-normalize`)
- [x] I am using the latest stable version of `ffmpeg` or a recent build from Git master

**Expected behavior**
I was expecting the cover art to be transferred from the source, I've been using same command for a long time to normalise to mp3 and listen on my mobile. Previously this did work but at some point recently it has stopped. I am not sure when if it was the update of the OS from 21.1 => 21.2 or ffmpeg from 4.4 (I think) => 6.0 

**Actual behavior**
I receive this error
```
WARNING: The chosen output extension mp3 does not support video/cover art. It will be disabled
```

**Command**

```
ffmpeg-normalize -f *.m4a -v -nt peak -t0 -c:a libmp3lame -e='-q:a 5' -of . -ext mp3
```

Any output you get when running the command with the `--debug` flag:

```
ffmpeg-normalize --debug -f *.m4a -v -nt peak -t0 -c:a libmp3lame -e='-q:a 5' -of . -ext mp3
DEBUG: Running command: ['/home/ian/bin/ffmpeg', '-filters']
DEBUG: Parsing streams of S2023E22 - Bahija Jallal - The biotech revolution in cancer therapies.m4a
DEBUG: Running command: ['/home/ian/bin/ffmpeg', '-i', 'S2023E22 - Bahija Jallal - The biotech revolution in cancer therapies.m4a', '-c', 'copy', '-t', '0', '-map', '0', '-f', 'null', '/dev/null']
DEBUG: Stream parsing command output:
DEBUG: ffmpeg version n6.0-ffmpeg-windows-build-helpers Copyright (c) 2000-2023 the FFmpeg developers
  built with gcc 11 (Ubuntu 11.3.0-1ubuntu1~22.04)
  configuration: --pkg-config=pkg-config --pkg-config-flags=--static --extra-version=ffmpeg-windows-build-helpers --enable-version3 --disable-debug --disable-w32threads --enable-libv4l2 --enable-libxcb --enable-libxcb-shm --enable-libxcb-xfixes --enable-libxcb-shape --enable-libcaca --enable-gray --enable-fontconfig --enable-gmp --enable-libass --enable-libbluray --enable-libbs2b --enable-libfreetype --enable-libfribidi --enable-libgme --enable-libgsm --enable-libilbc --enable-libmodplug --enable-libmp3lame --enable-libopencore-amrnb --enable-libopencore-amrwb --enable-libopus --enable-libsnappy --enable-libsoxr --enable-libspeex --enable-libtheora --enable-libtwolame --enable-libvo-amrwbenc --enable-libvorbis --enable-libwebp --enable-libzimg --enable-libzvbi --enable-libmysofa --enable-libopenjpeg --enable-libopenh264 --enable-libvmaf --enable-libsrt --enable-libxml2 --enable-opengl --enable-libdav1d --enable-cuda-llvm --enable-gnutls --enable-libsvtav1 --enable-libvpx --enable-libaom --extra-libs=-lharfbuzz --extra-libs=-lm --extra-libs=-lmpg123 --extra-libs=-lpthread --extra-cflags=-DLIBTWOLAME_STATIC --extra-cflags=-DMODPLUG_STATIC --extra-cflags=-DCACA_STATIC --enable-amf --disable-libmfx --enable-gpl --enable-frei0r --enable-librubberband --enable-libvidstab --enable-libx264 --enable-libx265 --enable-avisynth --enable-libaribb24 --enable-libxvid --enable-libdavs2 --enable-libxavs2 --extra-cflags='-mtune=generic' --extra-cflags=-O3 --enable-static --disable-shared --prefix=/home/ian/sources/ffmpeg-windows-build-helpers/sandbox/cross_compilers/native --enable-nonfree --enable-libfdk-aac
  libavutil      58.  2.100 / 58.  2.100
  libavcodec     60.  3.100 / 60.  3.100
  libavformat    60.  3.100 / 60.  3.100
  libavdevice    60.  1.100 / 60.  1.100
  libavfilter     9.  3.100 /  9.  3.100
  libswscale      7.  1.100 /  7.  1.100
  libswresample   4. 10.100 /  4. 10.100
  libpostproc    57.  1.100 / 57.  1.100
[mov,mp4,m4a,3gp,3g2,mj2 @ 0x555afbe2b100] stream 0, timescale not set
Input #0, mov,mp4,m4a,3gp,3g2,mj2, from 'S2023E22 - Bahija Jallal - The biotech revolution in cancer therapies.m4a':
  Metadata:
    major_brand     : M4A 
    minor_version   : 512
    compatible_brands: M4A isomiso2
    media_type      : 1
    copyright       : 2023 British Broadcasting Corporation, all rights reserved
    title           : Bahija Jallal on the biotech revolution in cancer therapies
    artist          : BBC Radio 4
    album_artist    : BBC Radio
    album           : The Life Scientific
    grouping        : Factual,Science & Nature,Science & Technology,Podcasts,Life Stories
    composer        : BBC Sounds
    genre           : Factual
    comment         : Bahija Jallal on the biotech revolution in cancer therapies.
    date            : 2023-09-05T09:00:00+01:00
    lyrics          : Some of the most complex medicines available today are made from living cells or organisms - these treatments are called biopharmaceuticals and in this episode of The Life Scientific Dr Bahija Jallal, CEO of Immunocore, shares her story of leaving her hom 
                    :  
                    : She tells Professor Jim Al-Khalili that she has always found herself ahead of the curve. When she began in oncology, the study of cancer, the common treatment was chemotherapy which attacked all the cells in an affected area. Her first studies into cancer 
                    :  
                    : It was an early step in what became known as targeted cancer therapies, and it set Bahjia on course for a career dedicated to developing innovative drugs to improve cancer patients' lives. Through a deep understanding of the science and a resolute commitm 
                    :  
                    : Presented by Professor Jim Al-Khalili 
                    : Produced by Tom Bonnett 
                    :  
                    : PLAY: https://www.bbc.co.uk/programmes/m001q670 
                    :  
                    : INFO: https://www.bbc.co.uk/programmes/m001q670
  Duration: 00:28:17.49, start: 0.000000, bitrate: 323 kb/s
  Stream #0:0[0x1](eng): Audio: aac (LC) (mp4a / 0x6134706D), 48000 Hz, stereo, fltp, 320 kb/s (default)
    Metadata:
      handler_name    : SoundHandler
      vendor_id       : [0][0][0][0]
  Stream #0:1[0x0]: Video: mjpeg (Baseline), yuvj420p(pc, bt470bg/unknown/unknown), 1920x1080 [SAR 72:72 DAR 16:9], 90k tbr, 90k tbn (attached pic)
Output #0, null, to '/dev/null':
  Metadata:
    major_brand     : M4A 
    minor_version   : 512
    compatible_brands: M4A isomiso2
    media_type      : 1
    copyright       : 2023 British Broadcasting Corporation, all rights reserved
    title           : Bahija Jallal on the biotech revolution in cancer therapies
    artist          : BBC Radio 4
    album_artist    : BBC Radio
    album           : The Life Scientific
    grouping        : Factual,Science & Nature,Science & Technology,Podcasts,Life Stories
    composer        : BBC Sounds
    genre           : Factual
    comment         : Bahija Jallal on the biotech revolution in cancer therapies.
    date            : 2023-09-05T09:00:00+01:00
    lyrics          : Some of the most complex medicines available today are made from living cells or organisms - these treatments are called biopharmaceuticals and in this episode of The Life Scientific Dr Bahija Jallal, CEO of Immunocore, shares her story of leaving her hom 
                    :  
                    : She tells Professor Jim Al-Khalili that she has always found herself ahead of the curve. When she began in oncology, the study of cancer, the common treatment was chemotherapy which attacked all the cells in an affected area. Her first studies into cancer 
                    :  
                    : It was an early step in what became known as targeted cancer therapies, and it set Bahjia on course for a career dedicated to developing innovative drugs to improve cancer patients' lives. Through a deep understanding of the science and a resolute commitm 
                    :  
                    : Presented by Professor Jim Al-Khalili 
                    : Produced by Tom Bonnett 
                    :  
                    : PLAY: https://www.bbc.co.uk/programmes/m001q670 
                    :  
                    : INFO: https://www.bbc.co.uk/programmes/m001q670
    encoder         : Lavf60.3.100
  Stream #0:0(eng): Audio: aac (LC) (mp4a / 0x6134706D), 48000 Hz, stereo, fltp, 320 kb/s (default)
    Metadata:
      handler_name    : SoundHandler
      vendor_id       : [0][0][0][0]
  Stream #0:1: Video: mjpeg (Baseline), yuvj420p(pc, bt470bg/unknown/unknown), 1920x1080 [SAR 72:72 DAR 16:9], q=2-31, 90k tbr, 90k tbn (attached pic)
Stream mapping:
  Stream #0:0 -> #0:0 (copy)
  Stream #0:1 -> #0:1 (copy)
Press [q] to stop, [?] for help
frame=    0 fps=0.0 q=-1.0 Lsize=N/A time=-577014:32:22.77 bitrate=N/A speed=N/A    /s speed=N/A    
video:0kB audio:0kB subtitle:0kB other streams:0kB global headers:0kB muxing overhead: unknown

DEBUG: Found duration: 1697.049 s
DEBUG: Found audio stream at index 0
DEBUG: Found video stream at index 1
INFO: Normalizing file S2023E22 - Bahija Jallal - The biotech revolution in cancer therapies.m4a (1 of 1)
DEBUG: Running normalization for S2023E22 - Bahija Jallal - The biotech revolution in cancer therapies.m4a
DEBUG: Parsing normalization info for S2023E22 - Bahija Jallal - The biotech revolution in cancer therapies.m4a
INFO: Running first pass astats filter for stream 0
DEBUG: Running command: ['/home/ian/bin/ffmpeg', '-hide_banner', '-y', '-i', 'S2023E22 - Bahija Jallal - The biotech revolution in cancer therapies.m4a', '-filter_complex', '[0:0]astats=measure_overall=Peak_level+RMS_level:measure_perchannel=0', '-vn', '-sn', '-f', 'null', '/dev/null']
DEBUG: ffmpeg output: [mov,mp4,m4a,3gp,3g2,mj2 @ 0x55ae407a29c0] stream 0, timescale not set
Input #0, mov,mp4,m4a,3gp,3g2,mj2, from 'S2023E22 - Bahija Jallal - The biotech revolution in cancer therapies.m4a':
Metadata:
major_brand     : M4A
minor_version   : 512
compatible_brands: M4A isomiso2
media_type      : 1
copyright       : 2023 British Broadcasting Corporation, all rights reserved
title           : Bahija Jallal on the biotech revolution in cancer therapies
artist          : BBC Radio 4
album_artist    : BBC Radio
album           : The Life Scientific
grouping        : Factual,Science & Nature,Science & Technology,Podcasts,Life Stories
composer        : BBC Sounds
genre           : Factual
comment         : Bahija Jallal on the biotech revolution in cancer therapies.
date            : 2023-09-05T09:00:00+01:00
lyrics          : Some of the most complex medicines available today are made from living cells or organisms - these treatments are called biopharmaceuticals and in this episode of The Life Scientific Dr Bahija Jallal, CEO of Immunocore, shares her story of leaving her hom
:
: She tells Professor Jim Al-Khalili that she has always found herself ahead of the curve. When she began in oncology, the study of cancer, the common treatment was chemotherapy which attacked all the cells in an affected area. Her first studies into cancer
:
: It was an early step in what became known as targeted cancer therapies, and it set Bahjia on course for a career dedicated to developing innovative drugs to improve cancer patients' lives. Through a deep understanding of the science and a resolute commitm
:
: Presented by Professor Jim Al-Khalili
: Produced by Tom Bonnett
:
: PLAY: https://www.bbc.co.uk/programmes/m001q670
:
: INFO: https://www.bbc.co.uk/programmes/m001q670
Duration: 00:28:17.49, start: 0.000000, bitrate: 323 kb/s
Stream #0:0[0x1](eng): Audio: aac (LC) (mp4a / 0x6134706D), 48000 Hz, stereo, fltp, 320 kb/s (default)
Metadata:
handler_name    : SoundHandler
vendor_id       : [0][0][0][0]
Stream #0:1[0x0]: Video: mjpeg (Baseline), yuvj420p(pc, bt470bg/unknown/unknown), 1920x1080 [SAR 72:72 DAR 16:9], 90k tbr, 90k tbn (attached pic)
Stream mapping:
Stream #0:0 (aac) -> astats:default
astats:default -> Stream #0:0 (pcm_s16le)
Press [q] to stop, [?] for help
Output #0, null, to '/dev/null':
Metadata:
major_brand     : M4A
minor_version   : 512
compatible_brands: M4A isomiso2
media_type      : 1
copyright       : 2023 British Broadcasting Corporation, all rights reserved
title           : Bahija Jallal on the biotech revolution in cancer therapies
artist          : BBC Radio 4
album_artist    : BBC Radio
album           : The Life Scientific
grouping        : Factual,Science & Nature,Science & Technology,Podcasts,Life Stories
composer        : BBC Sounds
genre           : Factual
comment         : Bahija Jallal on the biotech revolution in cancer therapies.
date            : 2023-09-05T09:00:00+01:00
lyrics          : Some of the most complex medicines available today are made from living cells or organisms - these treatments are called biopharmaceuticals and in this episode of The Life Scientific Dr Bahija Jallal, CEO of Immunocore, shares her story of leaving her hom
:
: She tells Professor Jim Al-Khalili that she has always found herself ahead of the curve. When she began in oncology, the study of cancer, the common treatment was chemotherapy which attacked all the cells in an affected area. Her first studies into cancer
:
: It was an early step in what became known as targeted cancer therapies, and it set Bahjia on course for a career dedicated to developing innovative drugs to improve cancer patients' lives. Through a deep understanding of the science and a resolute commitm
:
: Presented by Professor Jim Al-Khalili
: Produced by Tom Bonnett
:
: PLAY: https://www.bbc.co.uk/programmes/m001q670
:
: INFO: https://www.bbc.co.uk/programmes/m001q670
encoder         : Lavf60.3.100
Stream #0:0: Audio: pcm_s16le, 48000 Hz, stereo, s16, 1536 kb/s
Metadata:
encoder         : Lavc60.3.100 pcm_s16le
video:0kB audio:318280kB subtitle:0kB other streams:0kB global headers:0kB muxing overhead: unknown
[Parsed_astats_0 @ 0x55ae408cac40] Overall
[Parsed_astats_0 @ 0x55ae408cac40] Peak level dB: -1.955217
[Parsed_astats_0 @ 0x55ae408cac40] RMS level dB: -21.052829
DEBUG: astats command output: [mov,mp4,m4a,3gp,3g2,mj2 @ 0x55ae407a29c0] stream 0, timescale not set
Input #0, mov,mp4,m4a,3gp,3g2,mj2, from 'S2023E22 - Bahija Jallal - The biotech revolution in cancer therapies.m4a':
Metadata:
major_brand     : M4A
minor_version   : 512
compatible_brands: M4A isomiso2
media_type      : 1
copyright       : 2023 British Broadcasting Corporation, all rights reserved
title           : Bahija Jallal on the biotech revolution in cancer therapies
artist          : BBC Radio 4
album_artist    : BBC Radio
album           : The Life Scientific
grouping        : Factual,Science & Nature,Science & Technology,Podcasts,Life Stories
composer        : BBC Sounds
genre           : Factual
comment         : Bahija Jallal on the biotech revolution in cancer therapies.
date            : 2023-09-05T09:00:00+01:00
lyrics          : Some of the most complex medicines available today are made from living cells or organisms - these treatments are called biopharmaceuticals and in this episode of The Life Scientific Dr Bahija Jallal, CEO of Immunocore, shares her story of leaving her hom
:
: She tells Professor Jim Al-Khalili that she has always found herself ahead of the curve. When she began in oncology, the study of cancer, the common treatment was chemotherapy which attacked all the cells in an affected area. Her first studies into cancer
:
: It was an early step in what became known as targeted cancer therapies, and it set Bahjia on course for a career dedicated to developing innovative drugs to improve cancer patients' lives. Through a deep understanding of the science and a resolute commitm
:
: Presented by Professor Jim Al-Khalili
: Produced by Tom Bonnett
:
: PLAY: https://www.bbc.co.uk/programmes/m001q670
:
: INFO: https://www.bbc.co.uk/programmes/m001q670
Duration: 00:28:17.49, start: 0.000000, bitrate: 323 kb/s
Stream #0:0[0x1](eng): Audio: aac (LC) (mp4a / 0x6134706D), 48000 Hz, stereo, fltp, 320 kb/s (default)
Metadata:
handler_name    : SoundHandler
vendor_id       : [0][0][0][0]
Stream #0:1[0x0]: Video: mjpeg (Baseline), yuvj420p(pc, bt470bg/unknown/unknown), 1920x1080 [SAR 72:72 DAR 16:9], 90k tbr, 90k tbn (attached pic)
Stream mapping:
Stream #0:0 (aac) -> astats:default
astats:default -> Stream #0:0 (pcm_s16le)
Press [q] to stop, [?] for help
Output #0, null, to '/dev/null':
Metadata:
major_brand     : M4A
minor_version   : 512
compatible_brands: M4A isomiso2
media_type      : 1
copyright       : 2023 British Broadcasting Corporation, all rights reserved
title           : Bahija Jallal on the biotech revolution in cancer therapies
artist          : BBC Radio 4
album_artist    : BBC Radio
album           : The Life Scientific
grouping        : Factual,Science & Nature,Science & Technology,Podcasts,Life Stories
composer        : BBC Sounds
genre           : Factual
comment         : Bahija Jallal on the biotech revolution in cancer therapies.
date            : 2023-09-05T09:00:00+01:00
lyrics          : Some of the most complex medicines available today are made from living cells or organisms - these treatments are called biopharmaceuticals and in this episode of The Life Scientific Dr Bahija Jallal, CEO of Immunocore, shares her story of leaving her hom
:
: She tells Professor Jim Al-Khalili that she has always found herself ahead of the curve. When she began in oncology, the study of cancer, the common treatment was chemotherapy which attacked all the cells in an affected area. Her first studies into cancer
:
: It was an early step in what became known as targeted cancer therapies, and it set Bahjia on course for a career dedicated to developing innovative drugs to improve cancer patients' lives. Through a deep understanding of the science and a resolute commitm
:
: Presented by Professor Jim Al-Khalili
: Produced by Tom Bonnett
:
: PLAY: https://www.bbc.co.uk/programmes/m001q670
:
: INFO: https://www.bbc.co.uk/programmes/m001q670
encoder         : Lavf60.3.100
Stream #0:0: Audio: pcm_s16le, 48000 Hz, stereo, s16, 1536 kb/s
Metadata:
encoder         : Lavc60.3.100 pcm_s16le
video:0kB audio:318280kB subtitle:0kB other streams:0kB global headers:0kB muxing overhead: unknown
[Parsed_astats_0 @ 0x55ae408cac40] Overall
[Parsed_astats_0 @ 0x55ae408cac40] Peak level dB: -1.955217
[Parsed_astats_0 @ 0x55ae408cac40] RMS level dB: -21.052829
INFO: Running second pass for S2023E22 - Bahija Jallal - The biotech revolution in cancer therapies.m4a
INFO: Adjusting stream 0 by 1.955217 dB to reach 0.0
WARNING: The chosen output extension mp3 does not support video/cover art. It will be disabled.
DEBUG: Running command: ['/home/ian/bin/ffmpeg', '-hide_banner', '-y', '-i', 'S2023E22 - Bahija Jallal - The biotech revolution in cancer therapies.m4a', '-filter_complex', '[0:0]volume=1.955217dB[norm0]', '-map_metadata', '0', '-map_metadata:s:a:0', '0:s:a:0', '-map_chapters', '0', '-map', '[norm0]', '-c:a', 'libmp3lame', '-c:s', 'copy', '-q:a', '5', '/tmp/tmpr7ufj5eu/out.mp3']
DEBUG: ffmpeg output: [mov,mp4,m4a,3gp,3g2,mj2 @ 0x55e83c860a00] stream 0, timescale not set
Input #0, mov,mp4,m4a,3gp,3g2,mj2, from 'S2023E22 - Bahija Jallal - The biotech revolution in cancer therapies.m4a':
Metadata:
major_brand     : M4A
minor_version   : 512
compatible_brands: M4A isomiso2
media_type      : 1
copyright       : 2023 British Broadcasting Corporation, all rights reserved
title           : Bahija Jallal on the biotech revolution in cancer therapies
artist          : BBC Radio 4
album_artist    : BBC Radio
album           : The Life Scientific
grouping        : Factual,Science & Nature,Science & Technology,Podcasts,Life Stories
composer        : BBC Sounds
genre           : Factual
comment         : Bahija Jallal on the biotech revolution in cancer therapies.
date            : 2023-09-05T09:00:00+01:00
lyrics          : Some of the most complex medicines available today are made from living cells or organisms - these treatments are called biopharmaceuticals and in this episode of The Life Scientific Dr Bahija Jallal, CEO of Immunocore, shares her story of leaving her hom
:
: She tells Professor Jim Al-Khalili that she has always found herself ahead of the curve. When she began in oncology, the study of cancer, the common treatment was chemotherapy which attacked all the cells in an affected area. Her first studies into cancer
:
: It was an early step in what became known as targeted cancer therapies, and it set Bahjia on course for a career dedicated to developing innovative drugs to improve cancer patients' lives. Through a deep understanding of the science and a resolute commitm
:
: Presented by Professor Jim Al-Khalili
: Produced by Tom Bonnett
:
: PLAY: https://www.bbc.co.uk/programmes/m001q670
:
: INFO: https://www.bbc.co.uk/programmes/m001q670
Duration: 00:28:17.49, start: 0.000000, bitrate: 323 kb/s
Stream #0:0[0x1](eng): Audio: aac (LC) (mp4a / 0x6134706D), 48000 Hz, stereo, fltp, 320 kb/s (default)
Metadata:
handler_name    : SoundHandler
vendor_id       : [0][0][0][0]
Stream #0:1[0x0]: Video: mjpeg (Baseline), yuvj420p(pc, bt470bg/unknown/unknown), 1920x1080 [SAR 72:72 DAR 16:9], 90k tbr, 90k tbn (attached pic)
Stream mapping:
Stream #0:0 (aac) -> volume:default
volume:default -> Stream #0:0 (libmp3lame)
Press [q] to stop, [?] for help
Output #0, mp3, to '/tmp/tmpr7ufj5eu/out.mp3':
Metadata:
major_brand     : M4A
minor_version   : 512
compatible_brands: M4A isomiso2
media_type      : 1
TCOP            : 2023 British Broadcasting Corporation, all rights reserved
TIT2            : Bahija Jallal on the biotech revolution in cancer therapies
TPE1            : BBC Radio 4
TPE2            : BBC Radio
TALB            : The Life Scientific
TIT1            : Factual,Science & Nature,Science & Technology,Podcasts,Life Stories
TCOM            : BBC Sounds
TCON            : Factual
comment         : Bahija Jallal on the biotech revolution in cancer therapies.
TDRC            : 2023-09-05T09:00:00+01:00
USLT            : Some of the most complex medicines available today are made from living cells or organisms - these treatments are called biopharmaceuticals and in this episode of The Life Scientific Dr Bahija Jallal, CEO of Immunocore, shares her story of leaving her hom
:
: She tells Professor Jim Al-Khalili that she has always found herself ahead of the curve. When she began in oncology, the study of cancer, the common treatment was chemotherapy which attacked all the cells in an affected area. Her first studies into cancer
:
: It was an early step in what became known as targeted cancer therapies, and it set Bahjia on course for a career dedicated to developing innovative drugs to improve cancer patients' lives. Through a deep understanding of the science and a resolute commitm
:
: Presented by Professor Jim Al-Khalili
: Produced by Tom Bonnett
:
: PLAY: https://www.bbc.co.uk/programmes/m001q670
:
: INFO: https://www.bbc.co.uk/programmes/m001q670
TSSE            : Lavf60.3.100
Stream #0:0(eng): Audio: mp3, 48000 Hz, stereo, fltp
Metadata:
handler_name    : SoundHandler
vendor_id       : [0][0][0][0]
encoder         : Lavc60.3.100 libmp3lame
video:0kB audio:30742kB subtitle:0kB other streams:0kB global headers:0kB muxing overhead: 0.007637%
DEBUG: Moving temporary file from /tmp/tmpr7ufj5eu/out.mp3 to ./S2023E22 - Bahija Jallal - The biotech revolution in cancer therapies.mp3
DEBUG: Normalization finished
INFO: Normalized file written to ./S2023E22 - Bahija Jallal - The biotech revolution in cancer therapies.mp3

```

**Environment (please complete the following information):**
- [ ] Your operating system = Linux Mint 21.2
- [ ] Your Python version / distribution (`python3 --version` or `python --version`) = Python 3.10.12
- [ ] Your ffmpeg version (`ffmpeg -version`) = ffmpeg version n6.0-ffmpeg-windows-build-helpers Copyright (c) 2000-2023 the FFmpeg developers built with gcc 11 (Ubuntu 11.3.0-1ubuntu1~22.04)

**Comment by slhck** [comment]
I believe this was a regression introduced in https://github.com/slhck/ffmpeg-normalize/pull/199.
MP3 can handle video in the specific case of that "video" track being cover art.
I will fix this and provide a patch release.

**Comment by slhck** [comment]
Fixed in b42fce00548adcdcd1a3b45876d7af2d80f0dd5f and v1.27.7.
